### ISSUE TITLE
Support for Brotli encoding

### ISSUE THREAD

**Issue by manniL** [netlify/ask-netlify#24]
## About me

- **My name:** Alexander Lichter
- **I've used Netlify:** for most of my static sites!

## My question in a little more detail

I've transitioned several pages from Cloudflare + my own VPS to Netlify. So far, I'm really happy with the experience! Fast builds, no worries, easy setup.

The only thing I am missing (that Cloudflare or my own NGINX/Apache offered) is **Brotli compression** for static assets. [Brotli.pro](https://brotli.pro/) says that my pages on Netlify (eg. https://lichter.io) are not delivered with Brotli compression.

Is Brotli support planned? If yes, do you have a rough ETA? :thinking: 

## Anything other comments for Netlify

Thanks for all the work and effort :pray:

**Comment by polarathene** [comment]
27.50 on the video for those that are looking for it.

@philhawksworth Is there anywhere to track the progress for this feature, like an issue to subscribe to? 

If you're worried about concerns with added processing on the CDN side and that due to time to ensure you do that right, it may take a while and be lower priority to get out the door - could initial support serve pre-processed brotli `.br` files(eg, `data.json`, `data.json.br`) when available? At least, it seemed like the concern was due to encoding assets to brotli on the fly within the CDN layer?

**Comment by vickylance** [comment]
@philhawksworth  Is the Brotli support added yet?

**Comment by zeroby0** [comment]
@vickylance doesn't seem so. Run a Netlify hosted website through brotli.pro and see. Lichter.io is still being served through cloudflare.

**Comment by Illyism** [comment]
You can save bandwidths costs by shaving off about 25% bytes compared to gzip.

**Comment by vickylance** [comment]
@zeroby0 Yup I added brotli support to my website but its not picking up the brotli support yet but I deployed my website in  now.sh and it served my website out of the box as broti encoding.

**Comment by vickylance** [comment]
My website can save 10% in bandwidth cost upon initial test

**Comment by TonioGela** [comment]
Why this issue was closed without implementing the support?

**Comment by jgierer12** [comment]
@TonioGela because it was answered by the Netlify team. This is a Q&A repo, not a place for feature requests.

**Comment by polarathene** [comment]
Because the repo is not related to features for Netlify to implement?

> *A place to submit questions for Netlify to answer in tutorials, podcasts and blog posts https://ask.netlify.com*

**Comment by TonioGela** [comment]
Okay, my bad, sorry

**Comment by lucashfreitas** [comment]
+1.

**Comment by NullVoxPopuli** [comment]
Closed by accident?

**Comment by polarathene** [comment]
@NullVoxPopuli no, see my last comment. This isn't the place for requesting Netlify features.

**Comment by manniL** [comment]
BTW: Brotli is now available on Netlify 🔥

https://www.netlify.com/blog/2020/05/20/gain-instant-performance-boosts-as-brotli-comes-to-netlify-edge/

**Comment by MarcelGerber** [comment]
This is awesome! 🔥

Yet, it would be even greater to have a way to compress the most-accessed files with a higher compression level: For one of our most-accessed JS files, Netlify's compression comes down to 430KB (gzip) vs 414KB (brotli).
Yet a simple `npx brotli-max file.js` results in just 339KB, which is a saving of about 20%!

Seeing that this file is accessed several million times a month, it would be great to be able to use these additional savings. The userspace solution suggested above would be great for that - we could create a `file.js.br` optimized version for the most-accessed files and Netlify could then use that version for its Brotli responses.
